spec/system/kudo_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo test', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+
+  it 'crud kudo' do
+    visit root_path
+    click_link 'Login page'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: employee.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
+
+    click_link 'New Kudo'
+    fill_in 'Title', with: 'Title test1'
+    fill_in 'Content', with: 'Content Test1'
+    click_button 'Create Kudo'
+
+    expect(page).to have_content 'Kudo was successfully created.'
+    expect(page).to have_content 'Kudos'
```

Comment:
Consider checking if newly created kudo's content and title are also present on the page